docs/packages/pkg/Sqlpp11.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    sqlpp11
```

Comment:
> packages/pkg/Sqlpp11.rst:11:Sqlpp:["Slop", "Scalp", "Slap", "Slip", "Slippy", "Sloop", "Slope", "Sloppy", "Scallop", "Clop", "Glop", "Scalper", "Skip", "Sleep", "Slurp", "Skippy", "Scalps", "Sculpt", "Clap", "Clip", "Gulp", "Kelp", "Scalp's"]

Should be `Sqlpp`.

Reply:
Is that a bug with the spell checker?
Strips numbers from words to be checked but not from the dictionary?

Reply:
> Is that a bug with the spell checker?

I guess so.